lib/utils/__tests__/checkAgainstRule.test.mjs

```diff
@@ -86,24 +86,25 @@ describe('checkAgainstRule', () => {
 		expect(warnings[0].column).toBe(1);
 	});
 
+	/** @todo remove once all fixable rules have been been migrated from context.fix */
 	it('outputs fixed code when provided a context object', async () => {
-		const root = postcss.parse('a { top: 0px; }');
+		const root = postcss.parse('a { font-weight: bold; }');
 		const context = { fix: true };
 
 		const warnings = [];
 
 		await checkAgainstRule(
 			{
-				ruleName: 'length-zero-no-unit',
-				ruleSettings: true,
+				ruleName: 'font-weight-notation',
```

Comment:
This rule will probably be the last one fixed.
It's neither included in recommended nor standard.